docs/docs-source/docs/modules/get-started-gcp-marketplace/pages/run-sample-application.adoc

```diff
@@ -0,0 +1,73 @@
+= Run a Sample Cloudflow Application on GCP Marketplace
+:toc:
+:toc-title: ON THIS PAGE
+:toclevels: 2
+
+== Setting up CLI
+
+With Cloudflow installed on a GKE cluster through the Marketplace, you can interact with Cloudflow via its `kubectl` plugin: `kubectl cloudflow`.
+If you haven't already, download the CLI from the https://github.com/lightbend/cloudflow/releases/tag/v2.0.0[Github release page].
+Unzip the downloaded archive and then put the output file `kubectl-cloudflow` under `/usr/local/bin/`.
+
+Then connect your local `kubectl` to the GKE cluster on which Cloudflow is installed. You can find the command to do that from the "Clusters" view:
+
+.Connect `kubectl` to GKE cluster
+image::connect-clusters.png[Connect kubectl to GKE cluster]
+
+It should look like the following:
+[source,bash]
+----
+$ gcloud container clusters get-credentials <cluster-name> --zone <zone> --project <gcp-project>
+----
+
+== Building and Deploying call-record-aggregator
+
+Next we'll deploy a sample Cloudflow application called https://github.com/lightbend/cloudflow/tree/master/examples/call-record-aggregator[call-record-aggregator] to the cluster.
+Clone the Cloudflow https://github.com/lightbend/cloudflow[Github repo] and `cd` into the `examples/call-record-aggregator` directory. Then run
+
+[source,bash]
+----
+$ sbt buildApp
+----
+
+to build the application. Once it's complete, you should see at the end of the output a `kubectl cloudflow deploy` command to deploy your application. Run it:
+
+[source,bash]
+----
+$ kubectl cloudflow deploy call-record-aggregator.json
+----
+
+Once the deployment of the application starts, you can view the progress of the pods at `call-record-aggregator` namespace:
+
+[source,bash]
+----
+$ kubectl -n call-record-aggregator get pods -w
+----
+
+== Using Lightbend Console
+Lightbend Console is a suite of tools provided by Lightbend that enables observing and monitoring Kubernetes applications.
+To monitor the call-record-aggregator application you just deployed, first find the pod in the namespace where you deployed Cloudflow (by default the `default` namespace) whose name starts with `console-frontend-`.
```

Comment:
We should mention that we need to setup a port-fwd to get to the console. 
And refer (link) to the ingress setup doc for a more permanent solution.